features/step_definitions/claim_messages_steps.rb

```diff
@@ -19,6 +19,8 @@
 end
 
 Then(/^I should see my message at the bottom of the message list$/) do
+  sleep 1
```

Comment:
It has to wait for the AJAX to return the JS that reloads the page - but the actual page reload cannot be waited for, hence the sleep statement. Exploring ways of getting around it.